tools/clang/unittests/HLSL/ShaderOpTest.cpp

```diff
@@ -616,7 +616,7 @@ void ShaderOpTest::CreateResources() {
       }
 
       D3D12_SUBRESOURCE_DATA transferData[16];
-      UINT width = R.Desc.Width;
+      UINT width = (UINT)R.Desc.Width;
```

Comment:
TIL that the Width param of D3D12_RESOURCE_DESC is UINT64 while the Height is UINT and that makes the little-minded hobgoblin of consistency inside me sad 😞